.github/workflows/linter.yml

```
@@ -21,13 +21,29 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v6
         with:
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)